app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,14 @@
+class CommentsController < ApplicationController
+  def new
+    @comments = Comment.new
+  end
+
```

Comment:
Great job @NahnahAJ 🤝

- [ ] [Optional] You may also add a params method to the comments_controller. You can learn more about strong parameters by following this [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides).